Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -970,25 +970,35 @@ extension TargetBuildDescription {
 extension LLBuildManifestBuilder {
     private func createProductCommand(_ buildProduct: ProductBuildDescription) throws {
         let cmdName = try buildProduct.product.getCommandName(config: self.buildConfig)
+        let outputNode: Node
 
         switch buildProduct.product.type {
         case .library(.static):
+            outputNode = try .file(buildProduct.binaryPath)
             try self.manifest.addShellCmd(
                 name: cmdName,
                 description: "Archiving \(buildProduct.binaryPath.prettyPath())",
                 inputs: buildProduct.objects.map(Node.file),
-                outputs: [.file(buildProduct.binaryPath)],
+                outputs: [outputNode],
                 arguments: try buildProduct.archiveArguments()
             )
+        case .library(.automatic):
+            outputNode = .virtual(cmdName)
+            self.manifest.addPhonyCmd(
+                name: cmdName,
+                inputs: buildProduct.objects.map(Node.file),
+                outputs: [outputNode]
+            )
```

Comment:
This is the core change, we create a command now to produce all object files for an automatic product.